src/features/embalm/stepContent/components/SuccessfulCreateSarcophagus.tsx

```diff
@@ -12,87 +15,168 @@ export function SuccessfulCreateSarcophagus({
 }: SuccessData) {
   const navigate = useNavigate();
   const dispatch = useDispatch();
+  const networkConfig = useNetworkConfig();
 
   const goToDashboard = () => {
     dispatch(goToStep(Step.NameSarcophagus));
     navigate('/dashboard');
   };
 
+  const boxBorder = '0.5px solid';
+
+  const { onCopy: copyShardsId, hasCopied: hasCopiedShardsId } = useClipboard(
+    successEncryptedShardsTxId,
+    { timeout: 5000 }
+  );
+  const { onCopy: copyFileId, hasCopied: hasCopiedFileId } = useClipboard(
+    successSarcophagusPayloadTxId,
+    { timeout: 5000 }
+  );
+
+  const copyIcon = (hasCopiedFlag: boolean, onClick: () => void) =>
+    hasCopiedFlag ? (
+      <CheckIcon mt={4} />
+    ) : (
+      <CopyIcon
+        mt={4}
+        cursor="pointer"
+        bgColor={'transparent'}
+        color="white"
+        aria-label="copy"
+        onClick={onClick}
+      />
+    );
+
+  const arweaveTxId = (txId: string) => (
+    <Text
+      width="50%"
+      fontSize="10px"
+      color="brand.700"
+      textAlign="right"
+      mt={4}
+    >
+      {`${txId.slice(0, 5)}...${txId.slice(-4)}`}
+    </Text>
+  );
+
   return (
     <Flex
-      mt={6}
       direction="column"
       justifyContent="center"
       alignItems="center"
     >
+      {/*
+       * SUCCESS MESSAGE
+       */}
       <Heading
         fontSize="xl"
-        mb={4}
-      >
-        Sarcophagus Creation Successful!
-      </Heading>
-      <Heading
-        fontSize="md"
-        my={4}
+        mx={5}
+        mb={6}
       >
-        Transaction Details
+        Sarcophagus Created Successfully
       </Heading>
+      <SuccessIcon
+        width={50}
+        height={50}
+        mb={6}
+      />
+
+      {/*
+       * DETAILS BOX AND CONTENT
+       */}
       <Flex
-        justifyContent="space-between"
-        wrap="wrap"
+        width={700}
+        direction="column"
       >
-        <Text
-          width="40%"
-          fontSize="sm"
-          mt={4}
-        >
-          Sarcophagus TX
-        </Text>
-        <Text
-          width="60%"
-          fontSize="sm"
-          textAlign="right"
-          mt={4}
-        >
-          {successSarcophagusTxId}
-        </Text>
-        <Text
-          width="40%"
-          fontSize="sm"
-          mt={4}
-        >
-          Arweave File TX
-        </Text>
-        <Text
-          width="60%"
-          fontSize="sm"
-          textAlign="right"
-          mt={4}
-        >
-          {successSarcophagusPayloadTxId}
-        </Text>
-        <Text
-          width="40%"
-          fontSize="sm"
-          mt={4}
+        <Heading
+          fontSize="md"
+          py="12px"
```

Comment:
I do think we should try to standardize where appropriate on `{5}` and `10px` spacing for padding and margins. Just for consistencies? I probably need to bring this up somewhere other than this comment.